src/yb/master/master_tserver.h

```diff
@@ -80,7 +80,9 @@ class MasterTabletServer : public tserver::TabletServerIf,
 
   std::vector<yb::util::WaitStateInfoPtr> GetThreadpoolWaitStates() const override;
 
-  std::vector<WaitStateInfoPB> ActiveUniverseHistory() const override;
+  void ActiveUniverseHistory(tserver::PgActiveUniverseHistoryResponsePB* resp) const override {}
```

Comment:
I think we should just add a very simple method to TabletServerIf like cql_server_messenger(); or cql_server() which returns null by default. In TServer, we'll have the CQL server register/update this to return it.

TServers/messenger shouldn't depend/know about what PB/request are created. Can have a method in /util which takes a messenger and populates the appropriate PB/response